firefox_ui_harness/runtests.py

```diff
@@ -16,7 +16,46 @@
 
 class ReleaseTestRunner(BaseMarionetteTestRunner):
     extra_prefs = {
-        "browser.tabs.remote.autostart": False,
+        'browser.tabs.remote.autostart': False,
+        'app.update.auto': False,
+        'app.update.enabled': False,
+        'browser.dom.window.dump.enabled': True,
+        'browser.newtab.url': 'about:blank',
+        'browser.newtabpage.enabled': False,
+        'browser.safebrowsing.enabled': False,
+        'browser.safebrowsing.malware.enabled': False,
+        'browser.search.update': False,
+        'browser.sessionstore.resume_from_crash': False,
+        'browser.shell.checkDefaultBrowser': False,
+        'browser.startup.page': 0,
+        'browser.tabs.animate': False,
+        'browser.tabs.warnOnClose': False,
+        'browser.tabs.warnOnOpen': False,
+        'browser.uitour.enabled': False,
+        'browser.warnOnQuit': False,
+        'datareporting.healthreport.service.enabled': False,
+        'datareporting.healthreport.uploadEnabled': False,
+        'datareporting.healthreport.documentServerURI': "http://%(server)s/healthreport/",
+        'datareporting.healthreport.about.reportUrl': "http://%(server)s/abouthealthreport/",
```

Comment:
Again, please check if we really need all of them if the service is disabled anyway. You could check via hg check-ins how added those prefs for the mochitest framework.